src/globals/scss/_spacing.scss

```diff
@@ -0,0 +1,12 @@
+// Measurements used across GOV.UK
+$one-quarter: $full-width / 4;
+$one-third: $full-width / 3;
+$half: $full-width / 2;
+$two-thirds: ($full-width) - ($one-third);
+$three-quarters: ($full-width) - ($one-quarter);
+
+$gutter: 30px;
```

Comment:
Would it be possible to only copy over the vars that are used?
That way we can get rid of the ones we don't need. 

These will need a `govuk-` prefix too, can we keep them in `vars` for now and we split them out later.